app/controllers/admin/events_controller.rb

```diff
@@ -92,7 +95,27 @@ def update
       end
     end
 
-    def create; end
+    def create
+      @url = admin_conference_program_events_path(@conference.short_title, @event)
+      @users = User.all.order(:name)
+      @languages = @program.languages_list
+      @event.submitter = current_user
+      @event.state = :confirmed
+
+      if @event.save
+        ahoy.track 'Event submission', title: 'New submission'
+        redirect_to admin_conference_program_events_path(@conference.short_title), notice: 'Event was successfully submitted.'
+      else
+        flash[:error] = "Could not submit proposal: #{@event.errors.full_messages.join(', ')}"
+        render action: 'new'
```

Comment:
Nevermind, I found a solution,
it happened because the **@event.speakers.pluck(:id)**  will not return the ids for not-yet-persisted relations.
replacing this code with **@event.speakers.map(&:id) solved** the issue.